lib/liveValidation/operationValidator.ts

```diff
@@ -75,11 +81,22 @@ export const validateSwaggerLiveRequest = async (
     }
     const startTimeToBuild = Date.now();
     validate = await loader.getRequestValidator(operation);
+    const elapsedTime = Date.now() - startTimeToBuild;
     if (logging) {
       logging(
-        `On-demand build request validator with DurationInMs:${Date.now() - startTimeToBuild}`,
+        `On-demand build request validator with DurationInMs:${elapsedTime}`,
```

Comment:
Is it possible to aggregate this metric in kusto?

Reply:
Yes, we can.